pkg/utils/watchChaosContainer.go

```diff
@@ -2,32 +2,57 @@ package utils
 
 import (
 	"time"
-
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+
+var (
+	//chaosPod holds the experiment pod spec
+	chaosPod = &corev1.Pod{}
+)
+
+
+// GetChaosPodName gets the name of the chaos experiment pod launched by the runner
```

Comment:
Comment is not fit a/c to method definition

Reply:
Updated!